.github/workflows/build-test.yml

```diff
@@ -0,0 +1,35 @@
+on:
```

Comment:
I would suggest to use the docker file from this PR https://github.com/terra-money/community-pool-vesting/pull/5/files. 

What do you think?